### PR TITLE
Add primary scripts

### PR DESCRIPTION
**Pull request by simoncozens**
This is for discussion. It applies the script `gftools-set-primary-script` defined [here](https://github.com/googlefonts/gftools/pull/620) to the catalogue, setting the primary script of all non-Latin fonts in the catalogue.

### CODE REVIEW

File: ofl/lohitbengali/METADATA.pb [hunk 1]
```diff
@@ -13,3 +13,4 @@ fonts {
   copyright: "Copyright 2011 Lohit Fonts Project contributors (psatpute@redhat.com). <http://fedorahosted.org/lohit> Licensed under the SIL Open Font License 1.1 (see file OFL.txt)"
 }
 subsets: "bengali"
+primary_script: "Beng"
```
Comment (RosaWagner) [inline ofl/lohitbengali/METADATA.pb:16]: The three LOHIT families are not in the live catalog, maybe the script should only use the family list from the live server.

Comment (simoncozens) [reply ofl/lohitbengali/METADATA.pb:16]: Is it a problem to add this information to non-live fonts?

Comment (RosaWagner) [reply ofl/lohitbengali/METADATA.pb:16]: We generate the lists using all modified directories path so yes cause they might be pushed by mistake.

**Comment by RosaWagner** [comment]
Due to conflicts and stuff to fix in the script, I close this PR. In term of method, would be good to do it per group of scripts to make it easier to check in sandbox.

File: ofl/lohitbengali/METADATA.pb (hunk 1 above)
Comment (davelab6) [reply ofl/lohitbengali/METADATA.pb:16]: I forgotten why lohit not added, is it no Latin?

**Pull request closed without merging**